### ISSUE TITLE
Blur isn't working

### ISSUE THREAD

**Issue by W1nKille3r** [WesselKroos/youtube-ambilight#103]
**Describe the bug**

**To Reproduce**
Idk why, but on my GTX 770 blur isn't working. I can't use amberlight propetly.

**Expected behavior**
Its keep blur on

**Screenshots**
https://ibb.co/LvB0VrD

**Desktop (please complete the following information):**
 - OS: Windows 10 64x
 - Browser Chrome
 - Version Latest from Chrome Store

**Comment by WesselKroos** [comment]
@W1nKille3r What version of Chrome are you using? You can copy the version from Chrome's "menu > help > about" window

**Comment by W1nKille3r** [comment]
> 
I just reinstalled chrome from 0.
So, its latest Version 97.0.4692.99 (Official Build) (64-bit)

**Comment by W1nKille3r** [comment]
> 

Yeah, its just make spreading. Change debanding doesn't affect at all.
I tryed to change hardware acceleration - no effect

**Comment by W1nKille3r** [comment]
> 

So in short: Blur / Removing Black Bars doesn't working.
Tryed to restart GPU, no effect

**Comment by WesselKroos** [comment]
@W1nKille3r Alright, that rules performance issues out. Did the blur work before or did you just install the extension today?

Can you check if the blur filter is applied at all in the html code?
1. On the YouTube video page press **F12**. This will open a **Chrome Devtools** window
2. Go to the **Elements** tab and check if the **style="filter: blur(..px);"** attribute exist on the **ambilight__filter** element:
![image](https://user-images.githubusercontent.com/31220528/151061891-91882538-5f2c-4040-9bc5-bad2bf520428.png)

Can you also check if there are any red errors or yellow warnings in the **Console** tab and make a screenshot?
For example:
![image](https://user-images.githubusercontent.com/31220528/151062682-d2c560d9-be5c-4668-bb16-5d818d2b227d.png)

**Comment by W1nKille3r** [comment]
> 

**Screen with filter: blur**
https://ibb.co/GvbcBSD

**Console**
https://ibb.co/yn2KPmT
https://ibb.co/4ZnwVm5

**Comment by W1nKille3r** [comment]
> 

So... I didn't think of the most banal thing. Check if there is a conflict between other extensions
It turned out that one of them interfered with the work. Now I'll try to find turning off the extensions one by one. It would seem the most obvious, but I'm so dummy xD

**Comment by WesselKroos** [comment]
I did not see any problems in the screenshots, the filter is correct in the Elements tab. So I'm curious which of the other extension causes the conflict.

**Comment by W1nKille3r** [comment]
The problem was completely invisible. I had an old version of **"Dark Reader" - Dark Retired**

I don't like to break my eyes and use it regularly. It turned out that I had it installed since 19? For some reason, the developers said because "your browser is old, download this version." And for two years I used considered embedded code. Which by the way bugged often.

I don’t know how it was possible to get so fucked up, but now I downloaded the latest Dark Reader and the problem with the blur disappeared.
